app/views/docs/authentication-oauth.phtml

```diff
@@ -0,0 +1,370 @@
+<p>
+    OAuth authentication allows users to log in using accounts from other popular services. 
+    This can be convenient for users because they can start using your app without creating a new account.
+    It can also be more secure, because the user has one less password that could become vulnerable.
+</p>
+
+<p>
+    When using OAuth to authenticate, the authentication request is initiated from the client application. 
+    The user is then redirected to an OAuth 2 provider to complete the authentication step, and finally, the user is redirected back to the client application.
+</p>
+
+<h2><a href="#configure" id="configure">Configure OAuth 2 Login</a></h2>
+<p>
+    Before using OAuth 2 login, you need to enable and configure an OAuth 2 login provider.
+</p>
+
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>Navigate to your Appwrite project</li>
+    <li>Navigate to <b>Auth</b> > <b>Settings</b></li>
+    <li>Find and open the OAuth provider.</li>
+    <li>In the OAuth 2 settings modal, use the toggle to enable the provider</li>
```

Comment:
We are missing steps:

- Create app in developer platform of your OAuth 2 provider
- Enter nessessary information from OAuth 2 provider app
- Configure redirect URL in your OAuth 2 provider developer platform. Set it to URL provided to you by modal in Appwrite Console

Reply:
Done